contract_api/registry.py

```diff
@@ -431,3 +432,24 @@ def curate_service(self, org_id, service_id, curated):
             service_repo.curate_service(org_id, service_id, 0)
         else:
             Exception("Invalid curation flag")
+
+    def get_service_media(self,org_id,service_id):
+        try:
+            query = """select `row_id`,url,`order`,file_type,asset_type,alt_text from service_media 
+            where service_id = %s and org_id = %s """
+            query_response = self.repo.execute(query,[service_id,org_id])
+            media = []
+            if len(query_response)==0:
+                return media
+            for response_item in query_response:
+                media.append({
+                    "row_id":response_item['row_id'],
+                    "url":response_item['url'],
+                    "file_type":response_item['file_type'],
+                    "order":response_item['order'],
+                    "alt_text":response_item['alt_text'],
+                    "asset_type":response_item['asset_type']
+                })
+            return media
+        except Exception as e:
```

Comment:
Will we get a slack alert in this case?

Reply:
I am using the existing api - get_service_for_given_org which already has slack handler. should I add separately for this method alone